build/scripts/build.ps1

```diff
@@ -275,24 +276,16 @@ function Deploy-VsixViaTool() {
 
     Write-Host "Uninstalling old Roslyn VSIX"
 
-    # Reverse the extension list so we uninstall in the proper order so that dependencies line up
```

Comment:
I would keep this in. The code will uninstall nothing (because extensions are all uninstalled by the new code) but will also force MEF composition before install happens.